clevercloud/review_app_after_succes.sh

```diff
@@ -14,6 +14,11 @@ fi
 echo "Activate virtualenv"
 source $HOME/venv/bin/activate
 
+echo "install requirements"
+pip install factory-boy faker
```

Comment:
Faker vient avec `factory-boy`
```suggestion
pip install factory-boy
```